src/dotnet-test-nunit/TestRunner.cs

```diff
@@ -236,6 +236,12 @@ void SetupLabelOutput(TestExecutionListener listener)
                  ? _options.DisplayTestLabels.ToUpperInvariant()
                  : "ON";
 
+            listener.TestStarted += (sender, args) =>
+            {
+                if (labels == "ALL")
```

Comment:
@somdoron Ah! I see you are already doing what I suggested. :-) 

See my comment above about using test started as the place to write the label for "All"